packages/gatsby-plugin-sitemap/src/__tests__/gatsby-node.js

```diff
@@ -1,21 +1,30 @@
-jest.mock(`fs`)
+jest.mock(`sitemap`, () => {
+  return {
+    simpleSitemapAndIndex: jest.fn(),
+  }
+})
 
-const fs = require(`fs`)
 const path = require(`path`)
 const sitemap = require(`sitemap`)
-
 const { onPostBuild } = require(`../gatsby-node`)
```

Comment:
Shouldn't we test onPreInit too? Making sure joi fires the right errors

Reply:
Doing this kinda feels like we'd be testing Joi...though I suppose it'd also be testing our config and Joi objects...I'll work on that.